package.json

```diff
@@ -21,6 +21,7 @@
     "reset": "run-s reset:git reset:node-modules reset:yarn-lock configure",
     "configure": "yarn run init-registries && yarn install --frozen-lockfile --non-interactive && yarn run build:dev:backend",
     "install-yarn": "npm install --global yarn@1.22.17",
+    "corepack": "npm i -g corepack && corepack enable && corepack prepare yarn@stable --activate",
```

Comment:
@charellesandig Please call the npm script `enable-corepack` instead of just `corepack` so that the name hints at what it's doing. Having a noun for the name for something that modifies operating system level dependencies on the machine where the script runs is a bit of a misnomer.